includes/payment-methods/class-klarna-payment-method.php

```diff
@@ -34,29 +34,82 @@ public function __construct( $token_service ) {
 		$this->countries                    = [ 'US', 'GB', 'AT', 'DE', 'NL', 'BE', 'ES', 'IT', 'IE', 'DK', 'FI', 'NO', 'SE' ];
 		$this->limits_per_currency          = [
 			'USD' => [
-				'min' => 1000,
-				'max' => 500000,
-			], // Represents USD 10 - 5,000.
+				'US' => [
+					'min' => 0,
+					'max' => 1000000,
+				],
+			],
 			'GBP' => [
-				'min' => 1000,
-				'max' => 500000,
-			], // Represents GBP 10 - 5,000.
+				'UK' => [
+					'min' => 0,
+					'max' => 1150000,
+				],
+			],
 			'EUR' => [
-				'min' => 1000,
-				'max' => 500000,
-			], // Represents EUR 10 - 5,000.
+				'AT' => [
+					'min' => 1,
+					'max' => 1000000,
+				],
+				'BE' => [
+					'min' => 1,
+					'max' => 1000000,
+				],
+				'DE' => [
+					'min' => 1,
+					'max' => 1000000,
+				],
+				'NL' => [
+					'min' => 1,
+					'max' => 1000000,
+				],
+				'FI' => [
+					'min' => 0,
+					'max' => 1000000,
+				],
+				'ES' => [
+					'min' => 0,
+					'max' => 1000000,
+				],
+				'IE' => [
+					'min' => 0,
+					'max' => 400000,
+				],
+				'IT' => [
+					'min' => 0,
+					'max' => 1000000,
+				],
+				// TODO: set correct limits for next three.
+				'DK' => [
+					'min' => 100,
+					'max' => 10000000,
+				],
+				'NO' => [
+					'min' => 0,
+					'max' => 10000000,
+				],
+				'SE' => [
+					'min' => 0,
+					'max' => 15000000,
+				],
```

Comment:
We've agreed to focus on domestic currencies, so I believe we can address it in the next iteration. Do you agree, @kalessil ?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/7692/commits/55889907ed43af63da770dfbf501cd6e0c2b2072

Reply:
Thanks @KarlisJ !